src/kpi_app/app.py

```diff
@@ -277,7 +271,7 @@ def read_event_log_contents():
 merged_df_contents
 
 
-def Anzahl_erstellter_Inhalte(days=90, week=0, year=0, days2=0, interest="all"):
+def get_number_of_created_contents(days=90, week=0, year=0, days2=0, interest="all"):
```

Comment:
Since we started with English, we should keep it.
And functions should be in snake case in lowercase.